web/gds-admin-ui/src/pages/app/details/contact/Contact.js

```diff
@@ -0,0 +1,25 @@
+
+import React from 'react'
+import { Col, Row } from 'react-bootstrap';
+import { formatDisplayedData } from '../../../../utils';
+
+function Contact({ data, type }) {
+    return (
+        <>
+            <p className="fw-bold">{type} contact :</p>
```

Comment:
Minor typo (in English) - could we please remove the space between contact and :, e.g. `contact:</p>`?